src/client/Item/Items.ts

```diff
@@ -72,6 +81,72 @@ export class Items {
     return new QueryIterator(this.clientContext, query, options, fetchFunction, this.container.url);
   }
 
+  /**
+   * Create a `ChangeFeedIterator` to iterate over pages of changes
+   * @param partitionKey
+   * @param changeFeedOptions
+   */
+  public readChangeFeed(
+    partitionKey: string | number | boolean,
```

Comment:
Why is partitionKey a parameter here? It's part of changeFeedOptions, right?

Reply:
Intentional choice to make users think about partition key. We've been doing this for the new .NET SDK we have we overloads for all the Item APIs as well. @southpolesteve, do you agree?

Reply:
well, looks quite messy. Do we really have to go this way? :)

Reply:
PartitionKey will soon be required right? If there are one or two required things in a function call I like supporting the overload. Makes the beginner experience clearer.

Reply:
re: being messy - isn't gonna be messy since JavaScript doesn't officially support overloads, so we have to handle all those overloads with 1 implementation. Just a reality of JavaScript, unfortunately. Other option is to always make them required and pass undefined in, which is a worse customer experience.